### PR TITLE
Sanketa/sensitivity setup

### PR DESCRIPTION
**Pull request by skadam02**
sensitivity conditions set up

### CODE REVIEW

File: mixed_pond_experimental/sensitivity_setup.py [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+    T_wC = np.array(T_wC_vec)
+
+    df = pd.DataFrame(T_wC)
+    
+    df1= pd.concat([air_temp_data, df], axis = 1)
+    
+    df1.to_csv('Water_temp_daily.csv',index=False)
+
+    plt.plot(T_wC)
+    plt.plot(data['degC_avg'])
```
Comment (drewmresnick) [inline mixed_pond_experimental/sensitivity_setup.py:213]: My understanding here is that you are plotting the model results against the averaged temperatures from the input data - however is this the observed water temperature from the pond? I cannot remember exactly, however when I was making the plots I had a completely separate file for the observed air and water temperature. So I want to just confirm if this is the observed data or if this is the from the POWER dataset? @skadam02

File: mixed_pond_experimental/sensitivity_setup.py [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+Rh = 76.211
+volume = 6153.05 #m3
+area = 4047 #m2
+t = 24 #day or 24 hrs
+
+#open csv file using pandas to create pandas dataframe 
+
+air_temp_data = pd.read_csv("diurnal_air_temp_khulna_daily.csv") #this value is in kelvin
+data = pd.read_csv("input_data_sensitivity.csv")
+watertemp = pd.read_csv('Realtime_watertemp.csv')
```
Comment (drewmresnick) [inline mixed_pond_experimental/sensitivity_setup.py:38]: Or is this a file of the observed data? @skadam02

File: mixed_pond_experimental/sensitivity_setup.py [hunk 3]
```diff
@@ -0,0 +1,222 @@
...
+    df = pd.DataFrame(T_wC)
+    
+    df1= pd.concat([air_temp_data, df], axis = 1)
+    
+    df1.to_csv('Water_temp_daily.csv',index=False)
+
+    plt.plot(T_wC)
+    plt.plot(data['degC_avg'])
+    plt.plot(watertemp['day_avg'])
+    plt.show()
```
Comment (drewmresnick) [inline mixed_pond_experimental/sensitivity_setup.py:215]: Can you add the legend with labels to the plot?

`plt.plot(watertemp['day_avg'], label="observed water temp")`
`plt.legend()`
`plt.show()`

**Review by drewmresnick — CHANGES_REQUESTED** [review]

**Comment by skadam02** [comment]
making changes per comments

**Pull request closed without merging**